connectivity/check/action.go

```diff
@@ -510,197 +549,198 @@ func (a *Action) GetIngressRequirements(p FlowParameters) []filters.FlowSetRequi
 	return []filters.FlowSetRequirement{ingress}
 }
 
-var errNeedMoreFlows = errors.New("Required flows not found yet")
-
-// ValidateFlows retrieves the flow pods of the specified pod and validates
-// that all filters find a match. On failure, t.Fail() is called.
-func (a *Action) ValidateFlows(ctx context.Context, pod, podIP string, reqs []filters.FlowSetRequirement) {
-	if a.test.ctx.params.FlowValidation == FlowValidationModeDisabled {
-		a.Logf("📄 Skipping flow validation for pod %s", pod)
-		return
+// followFlows starts a long-poll against Hubble for receiving all flow logs
+// pertaining to the Action's Pod until ctx is canceled.
+// Signals on the ready channel when the listener is ready to report traffic.
+// Returns error if any Hubble nodes return errors or are unavailable, or if
+// anything unexpected occurs during the follow operation.
+func (a *Action) followFlows(ctx context.Context, ready chan bool) error {
+	// Need a Hubble client to receive flows.
+	hubbleClient := a.test.ctx.HubbleClient()
+	if hubbleClient == nil {
+		return nil
```

Comment:
Shouldn't this be an error? If not, would you mind adding a comment explaining why returning `nil` here is OK?

Reply:
yeah i think it would be clearer to do something like:
```
if !a.test.ctx.params.Hubble {
  // hubble is disabled.
  return nil
}